### DISCUSSION TITLE
Alpha 3.2

### DISCUSSION THREAD

**Discussion by pavakhov** [audacity/audacity#3107] [Development]
It would be helpful to test/access a compiled binary for Windows and OS x, the recent version that uses the libmpg123 library would be useful.  The alpha version currently available is 3.0.3.
Much thanks

**Comment by petersampsonaudacity** [comment]
In GitHub here, click on the "Actions" tab - and download the latest "master" build

_If you want to be really bold and adventurous you can download and test the branches of the various developers_

**Comment by elvey** [comment]
I found the above instructions unfollowable today, though I've downloaded an alpha before; I got lost in the logs.  I finally figured it out again; _here are more detailed instructions_:

**How to download the latest alpha builds of Audacity** (currently Audacity 3.2.0 alpha 20220621+f25a787) : 
1. In GitHub here, click on the "Actions" tab (or open https://github.com/audacity/audacity/actions)
2. Click on the _title text_ next to the first green checkbox on a line indicated as a "master" build.
3. Click on the number under "Artifacts" or scroll down to see the "artifacts". Currently, this can be reached at https://github.com/audacity/audacity/actions/runs/2538244935#artifacts but the URL is changing very often.
5. Here, you can click to download the latest "master" build for the platform you want.  For me it's the 
audacity-macOS-3.2.0-alpha-...-arm64 build listed fourth. 
6. For me this downloads https://github.com/audacity/audacity/suites/7031104158/artifacts/276563540 but again the URL is changing very often.
7. I'm not sure why this take so much work; I can't remember it being such a challenge with other apps.

**Reply by LWinterberg** [reply]
you now can go to https://www.audacityteam.org/nightly instead
